activemodel/lib/active_model/type/boolean.rb

```diff
@@ -23,6 +23,8 @@ class Boolean < Value
         "FALSE", :FALSE,
         "off", :off,
         "OFF", :OFF,
+        "no", :no,
+        "NO", :NO,
```

Comment:
Doesn't this introduce the Norway ('NO' is the ISO 3166 country code) problem YAML has? Also see https://hitchdev.com/strictyaml/why/implicit-typing-removed/

Reply:
Ok, that's interesting. I wasn't aware of this type of issues. In my case, we explicitly use ActiveModel::Type::Boolean in some places when processing CSV files, for example.

I guess the question would be, is this YAML issue a reason not to let AciveModel::Type::Boolean cast 'No' as a valid boolean? Also, is there something I could do to help mitigate the issue and still have the ability to cast it?